erpnext/hr/doctype/department_approver/department_approver.py

```diff
@@ -36,13 +36,18 @@ def get_approvers(doctype, txt, searchfield, start, page_len, filters):
 	if filters.get("doctype") == "Expense Claim" and employee.expense_approver:
 		approvers.append(frappe.db.get_value("User", employee.expense_approver, ['name', 'first_name', 'last_name']))
 
+	if filters.get("doctype") == "Shift Request" and employee.shift_request_approver:
+		approvers.append(frappe.db.get_value("User", employee.shift_request_approver, ['name', 'first_name', 'last_name']))
 
 	if filters.get("doctype") == "Leave Application":
 		parentfield = "leave_approvers"
 		field_name = "Leave Approver"
-	else:
+	elif filters.get("doctype") == "Leave Application":
```

Comment:
shouldn't this be Expense Claim?
```suggestion
	elif filters.get("doctype") == "Expense Claim":
```

Reply:
done